README.md

```diff
@@ -43,6 +43,15 @@ HM Gutenberg Tools then exposes all functionality globally as `window.hm`. You c
 * `npm run watch` Watches for changes and builds development versions of the code.
 * `npm run lint` Lints your JS and fixes your code.
 
+## Releasing a new version.
+
+1. Update the version numbers in `plugin.php` and `package.json`.
+2. Add the changelog to the readme for the new version.
+3. Commit your changes to master and push. 
+4. Run the bash script: `./release.sh v1.2.3`
+
+The script will sync the the build branch with master, build assets and commit the changes, and publish a new tagged release.
```

Comment:
This is a nitpick, but I want to mention that this script does **not** create a new _release_ on GitHub. It only adds a Git tag (which will also show on GitHub, of course).

If the intention is to only use tags (and not GitHub releases), maybe "release" as noun should be replaced by tag or version. Releasing a new version is, of course, accurate.

So, how about this:

```suggestion
The script will sync the the build branch with master, build assets and commit the changes, and publish a new tagged version.
```